BlueprintUI/Sources/Element/ComposedElement.swift

```diff
@@ -0,0 +1,22 @@
+import CoreGraphics
+
+/// An element built out of other elements.
+///
+/// Similar in purpose to `ProxyElement`, but the contents may change depending on the `Environment`.
+///
+/// - seealso: ProxyElement
+public protocol ComposedElement: Element {
```

Comment:
+1

Reply:
Yeah, naming is hard. This has a very similar role to ProxyElement, so I tried to come up with a similar name, but maybe that's just confusing. If ProxyElement didn't exist, I think `ComposedElement` would be a good name.

I avoided `EnvironmentElement` because I think it's unclear whether that means it does something to the environment or consumes it or something else.

The main difference between this and ProxyElement is that this lets you dynamically create an element, while ProxyElement is static. Perhaps `DynamicProxyElement` is better?

Reply:
I wound up renaming this to `DynamicElement`, which I think distinguishes it better?